From e396e09f2f151066df908660d86d9dc8dc3ba2ca Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:14:19 -0400
Subject: [ethernet] be2net: use "if (!foo)" test style

Message-id: <1410192881-30702-57-git-send-email-ivecera@redhat.com>
Patchwork-id: 92442
O-Subject: [RHEL7.1 PATCH 56/78] be2net: use "if (!foo)" test style
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit ddf1169fec57c68f9ab561ac818b02aae95943b4
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Thu Jul 17 16:20:28 2014 +0530

    be2net: use "if (!foo)" test style

    Replace "if (foo == NULL)" statements with "if (!foo)" to be consistent
    across the driver.

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index 1214bfc..4213c70 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -2030,7 +2030,7 @@ static void be_rx_cq_clean(struct be_rx_obj *rxo)
   */
  for (;;) {
   rxcp = be_rx_compl_get(rxo);
-  if (rxcp == NULL) {
+  if (!rxcp) {
    if (lancer_chip(adapter))
     break;
 
@@ -2937,7 +2937,7 @@ static int be_setup_wol(struct be_adapter *adapter, bool enable)
  cmd.size = sizeof(struct be_cmd_req_acpi_wol_magic_config);
  cmd.va = dma_zalloc_coherent(&adapter->pdev->dev, cmd.size, &cmd.dma,
          GFP_KERNEL);
- if (cmd.va == NULL)
+ if (!cmd.va)
   return -ENOMEM;
 
  if (enable) {
@@ -4129,7 +4129,7 @@ lancer_fw_exit:
 static int be_get_ufi_type(struct be_adapter *adapter,
       struct flash_file_hdr_g3 *fhdr)
 {
- if (fhdr == NULL)
+ if (!fhdr)
   goto be_get_ufi_exit;
 
  if (skyhawk_chip(adapter) && fhdr->build[0] == '4')
@@ -4471,12 +4471,12 @@ static int be_map_pci_bars(struct be_adapter *adapter)
 
  if (BEx_chip(adapter) && be_physfn(adapter)) {
   adapter->csr = pci_iomap(adapter->pdev, 2, 0);
-  if (adapter->csr == NULL)
+  if (!adapter->csr)
    return -ENOMEM;
  }
 
  addr = pci_iomap(adapter->pdev, db_bar(adapter), 0);
- if (addr == NULL)
+ if (!addr)
   goto pci_map_err;
  adapter->db = addr;
 
@@ -4539,7 +4539,7 @@ static int be_ctrl_init(struct be_adapter *adapter)
  rx_filter->va = dma_zalloc_coherent(&adapter->pdev->dev,
          rx_filter->size, &rx_filter->dma,
          GFP_KERNEL);
- if (rx_filter->va == NULL) {
+ if (!rx_filter->va) {
   status = -ENOMEM;
   goto free_mbox;
  }
@@ -4588,7 +4588,7 @@ static int be_stats_init(struct be_adapter *adapter)
 
  cmd->va = dma_zalloc_coherent(&adapter->pdev->dev, cmd->size, &cmd->dma,
           GFP_KERNEL);
- if (cmd->va == NULL)
+ if (!cmd->va)
   return -ENOMEM;
  return 0;
 }
@@ -4810,7 +4810,7 @@ static int be_probe(struct pci_dev *pdev, const struct pci_device_id *pdev_id)
  pci_set_master(pdev);
 
  netdev = alloc_etherdev_mqs(sizeof(*adapter), MAX_TX_QS, MAX_RX_QS);
- if (netdev == NULL) {
+ if (!netdev) {
   status = -ENOMEM;
   goto rel_reg;
  }
-- 
1.7.1